source/_integrations/tado.markdown

```diff
@@ -180,3 +189,28 @@ script:
           entity_id: water_heater.hot_water
           time_period: "01:30:00"
 ```
+
+
+{% raw %}
+```yaml
+# Example automation to set temprature offset based on another thermostat value
+automation:
+    trigger:
+    - platform: state
+      entity_id:
+        - sensor.temp_sensor_room
+        - sensor.tado_temperature
+        
+    condition:
+    - condition: template
+      value_template: > 
+        {{ (states('sensor.tado_temperature')|float - states('sensor.temp_sensor_room')|float) > 0.5 }}
```

Comment:
Can we make the template less long but using multi-line templates? So it comes readable for the user.